ios/FluentUI/Divider/DividerModifiers.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
```

Comment:
there isn't anything? to modify? why we need this?

Reply:
https://github.com/microsoft/fluentui-apple/pull/1136#discussion_r939291944

Reply:
Resolved offline; let's remove it for now and add it back in when the time comes.